test/core/TakerOperations.t.sol

```diff
@@ -724,9 +729,9 @@ contract TakerOperationsTest is MangroveTest {
     }
     mkr.expect("mgv/tradeSuccess");
     // first two offers are at right price
-    uint takerWants = 0.1 ether;
-    uint takerGives = 2 * takerWants;
-    mgv.marketOrderByVolume{gas: 450_000}($(base), $(quote), takerWants, takerGives, true);
+    uint takerWants = 0.1 ether + 0.1 ether;
+    uint takerGives = 0.1 ether + 0.2 ether;
```

Comment:
This deviates from the pattern in the previous two tests.

I changed this because the old tests were unclear and one didn't actually make sense, I believe it was this one: The comment says the "first two offers are at right price" but that is not true now as evidenced by `offersExpectedTaken = 1` above.

I think this change should be reverted or it should be made clearer, what this test is supposed to test.

Reply:
addessed in cf54a2471e08dd93819e51782d782552aa7e1219